java/core/src/java/org/apache/orc/impl/SerializationUtils.java

```diff
@@ -270,8 +270,16 @@ public static BigInteger readBigInteger(InputStream input) throws IOException {
    * @return bits required to store value
    */
   public int findClosestNumBits(long value) {
-    final int numBits = 64 - Long.numberOfLeadingZeros(value);
-    return getClosestFixedBits(numBits);
+    return getClosestFixedBits(findNumBits(value));
+  }
+
+  /**
+   * Count the number of bits left after subtracting leading zeros
+   * @param value
+   * @return bits left after subtracting leading zeros
+   */
+  public int findNumBits(long value) {
```

Comment:
Do you have a reference for this naming, @guiyanakuang ? Can we have a more specific name?

Reply:
I've looked up some references and I still don't know what the name of the complement code to remove the high zeros is. How about `validNumBits`.

Reply:
Ya, it's tricky. Then, shall we make this `private`? In that case, we don't need to care much about naming.

Reply:
Yes, I should not expose this internal method.  Fix in 6fa50b2